src/package.json

```diff
@@ -1,15 +1,10 @@
 {
-  "name": "src",
+  "name": "bird-brain",
   "version": "1.5.0",
-  "description": "",
   "main": "views/main.html",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "window": {
     "title": "Bird Brain",
     "icon": "assets/icon.png",
-    "toolbar": true,
```

Comment:
`"toolbar"` is from NW.js v0.12.3 and before. It has been removed since v0.13.0 (2016)